react/features/av-moderation/middleware.js

```diff
@@ -35,21 +37,31 @@ import {
     participantApproved,
     participantPendingAudio
 } from './actions';
+import { ASKED_TO_UNMUTE_SOUND_ID } from './constants';
 import {
     isEnabledFromState,
     isParticipantApproved,
     isParticipantPending
 } from './functions';
+import { ASKED_TO_UNMUTE_FILE } from './sounds';
 
-const VIDEO_MODERATION_NOTIFICATION_ID = 'video-moderation';
-const AUDIO_MODERATION_NOTIFICATION_ID = 'audio-moderation';
-const CS_MODERATION_NOTIFICATION_ID = 'video-moderation';
+export const AUDIO_MODERATION_NOTIFICATION_ID = 'audio-moderation';
+export const VIDEO_MODERATION_NOTIFICATION_ID = 'video-moderation';
+export const CS_MODERATION_NOTIFICATION_ID = 'screensharing-moderation';
```

Comment:
middleware sounds like a weird place to define those. Maybe define them privately on the constants.js?